http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AzurePubsubHandler.scala

```diff
@@ -955,23 +955,7 @@ class AzurePubsubHandlerInterp[F[_]: Parallel](
     for {
       ctx <- ev.ask
       params = UpdateAKSAppParams(appId, appName, appChartVersion, workspaceId, cloudContext)
-      _ <- aksAlgebra.updateAndPollApp(params).adaptError { case e =>
```

Comment:
there is no need to `adaptError` on this inner function call and on the outer function's call in the top-level `LeoPubsubMessageSubscriber`.  Consolidated error handling to the top-level call since the requirements are the same for both google and azure messages.